src/Microsoft.VisualStudio.Web.CodeGeneration.Templating/Microsoft.VisualStudio.Web.CodeGeneration.Templating.csproj

```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Razor" Version="1.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Razor" Version="1.0.2" />
```

Comment:
This says 1.0.2
https://github.com/aspnet/Coherence-Patch/blob/rel/1.0.4/patchConfig.json#L373